sycl/source/detail/scheduler/commands.cpp

```diff
@@ -2535,12 +2535,19 @@ ur_result_t enqueueImpCommandBufferKernel(
       LocalSize = RequiredWGSize;
   }
 
+  ur_exp_command_buffer_desc_t CommandBufferDesc{};
+
+  Adapter->call<UrApiKind::urCommandBufferGetInfoExp>(
```

Comment:
Would stick in a wee single line comment saying a handles are only valid when a graph is updatable, to save somebody having to go lookup the UR spec.

Reply:
Good point, done!